netket/experimental/driver/vmc_srt.py

```diff
@@ -32,85 +32,169 @@
 from netket.utils.types import ScalarOrSchedule, Optimizer, PyTree
 from netket.vqs import MCState
 
-from jax.flatten_util import ravel_pytree
+from netket.utils.optional_deps import import_optional_dependency
+
+from netket.jax.sharding import shard_along_axis, all_gather
+
+nt = import_optional_dependency("neural_tangents", descr="neural_tangents")
+einops = import_optional_dependency("einops", descr="einops")
```

Comment:
This cannot be here. It must be inside a function.
If it's here, it will be required to load netket.

Reply:
I'm ok with adding einops as a dependency of netket, though. So just import it normally and add It as an official dependency